go.mod

```diff
@@ -8,28 +8,29 @@ require (
 	github.com/Azure/go-autorest/autorest/adal v0.9.22 // indirect
 	github.com/Azure/go-autorest/autorest/to v0.4.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/logr v1.2.4
 	github.com/golang/mock v1.6.0
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/onsi/ginkgo/v2 v2.8.1
-	github.com/onsi/gomega v1.26.0
-	github.com/openshift/api v0.0.0-20230213202419-42edf4f1d905
-	github.com/openshift/machine-api-operator v0.2.1-0.20230214202016-9d54a2c6e9a2
+	github.com/onsi/ginkgo/v2 v2.9.5
+	github.com/onsi/gomega v1.27.6
+	github.com/openshift/api v0.0.0-20230509100629-894b49f57a15
+	github.com/openshift/machine-api-operator v0.2.1-0.20230518185332-b256cd8c71ba
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.6.1
+	github.com/spf13/cobra v1.7.0
 	golang.org/x/crypto v0.6.0
+	golang.org/x/net v0.10.0 // indirect
 
-	// kube 1.26
-	k8s.io/api v0.26.1
-	k8s.io/apimachinery v0.26.1
-	k8s.io/client-go v0.26.1
-	k8s.io/klog/v2 v2.90.0
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
-	sigs.k8s.io/controller-runtime v0.14.4
-	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230214165931-2ebab48c37db
-	sigs.k8s.io/controller-tools v0.11.3
+	// kube 1.27
+	k8s.io/api v0.27.1
+	k8s.io/apimachinery v0.27.1
+	k8s.io/client-go v0.27.1
+	k8s.io/klog/v2 v2.100.1
+	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	sigs.k8s.io/controller-runtime v0.15.0-alpha.1
```

Comment:
We can now use `v0.15.0`

Reply:
Yeah, I have that change locally. Incorporating the fake client changes before pushing again.